fossunited/www/newsletter/index.html

```diff
@@ -1,176 +1,560 @@
+          Telegram Community</a
+        >.
+      </p>
     </div>
   </div>
 </section>
-
-<!-- Subject to display Change -->
 <section id="subscribed-page">
   <div id="success d-flex">
     <div id="success-wrapper">
       <div id="success-icon-div">
-        <svg class="success-icon container-center icon"><use href="#icon-solid-success"></use></svg>
+        <svg class="success-icon container-center icon">
+          <use href="#icon-solid-success"></use>
+        </svg>
       </div>
-      <div class="text-center ">
+      <div class="text-center">
         <p>
-        You are subscribed to our newsletter! You will be receiving updates every month about FOSS and FOSS United.
+          You are subscribed to our newsletter! You will be receiving updates
+          every month about FOSS and FOSS United.
         </p>
       </div>
     </div>
   </div>
 </section>
 
+<script src="/newsletter/index.js" type="text/javascript"></script>
 <style>
-#hero h1 {
-  font-size: 40px;
-}
-
-.news-icon img {
-  margin-bottom: -40px !important;
-}
-
-#hero p {
-  margin: auto;
-  width: auto;
-  max-width: 900px;
-  font-size: 20px;
-}
-
-.newsletter-icon img {
-  margin-bottom: -50px !important;
-}
-
-#newsletters-list {
-  margin: auto;
-}
-#email {
-  border: 1px solid black;
-  border-radius: 8px;
-  width: 360px;
-  height: 48px;
-  padding-left: 10px;
-}
-#news {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  gap: 12px;
-  padding: 24px 48px;
-  background: var(--gray-50, #F9FAFB);
-  color: #08b74f;
-  margin: 15px;
-}
-
-    #news a {
-      color: #08b74f;
-      max-width: 500px;
-    }
-
-    p {
-      padding: 0px 20px 0px 20px;
-    }
-    #subscribe {
-      padding: 12px;
+  .newsletter-home-page {
+    display: inline-flex;
+    padding: 6.6875rem 20rem 0rem 20rem;
+    flex-direction: column;
+    justify-content: flex-end;
+    margin: auto;
+    align-items: center;
+    gap: 2.5rem;
+    height: auto;
+    background: #f5f5f5;
+  }
+
+  .text-content {
+    text-align: center;
+  }
+
+  .text-content h1 {
+    color: hsl(var(--clr-code-night-900));
+    text-align: center;
+    font-family: Inter;
+    font-size: var(--text-4xl);
+    font-style: normal;
+    font-weight: 700;
+    line-height: 2.75rem;
+    /* 110% */
+    letter-spacing: -0.1rem;
+  }
+
+  .subscribe-section {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+  }
+
+  #email {
+    display: flex;
+    width: 25rem;
+    height: 2.5rem;
+    padding: 0.6875rem 0.75rem;
+    align-items: center;
+    gap: 0.875rem;
+    border-radius: 0.5rem;
+    border: hsl(var(--clr-gray-900));
+    background: hsl(var(--clr-gray-50));
+  }
+
+  #subscribe-button {
+    display: flex;
+    width: 9.125rem;
+    height: 2.5rem;
+    padding: 0.625rem 0.75rem;
+    justify-content: center;
+    align-items: center;
+    color: white;
+    gap: 0.5rem;
+    border-radius: 0.5rem;
+    background: hsl(var(--clr-foss-mint-500));
+  }
```

Comment:
just use the primary-button class on the subscribe button rather than doing all this